.github/workflows/build_and_publish.yaml

```diff
@@ -12,18 +12,19 @@ jobs:
       uses: actions/checkout@v2.3.3
     - run: |
         git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
-    - name: Build documentation  
-      run: |
-        sudo snap install yq
-        cd docs
-        make all
+        git fetch --depth=1 origin +refs/tags/*:refs/tags/*
     - name: Whitesource
       env:
         WHITESOURCE_PASSWORD: ${{ secrets.WHITESOURCE_PASSWORD }}
       run: |
-        git fetch --depth=1 origin +refs/tags/*:refs/tags/*
         cd core
         sbt "whitesourceCheckPolicies"
+        sbt "whitesourceUpdate"
```

Comment:
A single start of sbt would be enough `sbt "whitesourceCheckPolicies; whitesourceUpdate"`

Reply:
right, fixed!